siteResources.js

```diff
@@ -0,0 +1,25 @@
+export const urls = [
```

Comment:
Moved the config for the sites that we visit to a separate file. Should be able to just append new sites with the same keys and they will work in the script.